plugins/@grouparoo/app-templates/src/source/table/profileProperties.ts

```diff
@@ -95,7 +103,17 @@ export const getProfileProperties: GetProfilePropertiesMethod = ({
 
     for (const pk in responsesByPrimaryKey) {
       primaryKeysHash[pk].forEach((profileId) => {
-        responsesById[profileId] = responsesByPrimaryKey[pk];
+        // responsesById[profileId] = responsesByPrimaryKey[pk];
+        responsesById[profileId] = {};
```

Comment:
Is this clearing out of properties we've set the value of already intentional? For example, previously we had defaulted some values to `[0]` in

```ts
//default all entries to 0 to start... will be replaced later if there is a response
responsesById[profiles[i].id] = { [property.id]: [0] };
```